src/main/java/ch/njol/skript/effects/EffVisualEffect.java

```diff
@@ -47,8 +47,8 @@ public class EffVisualEffect extends Effect {
 
 	static {
 		Skript.registerEffect(EffVisualEffect.class,
```

Comment:
Accidentally included this

Reply:
Probably fixed by merging master @TheLimeGlass